### DISCUSSION TITLE
Proforma invoice

### DISCUSSION THREAD

**Discussion by vvvamik** [PrestaShop/PrestaShop#40205] [Ideas]
In Prestashop, the possibility of an proforma invoice is very missing. Is this feature planned? Thank you.

**Comment by khouloudbelguith** [comment]
Hi @vvvamik,

Thanks for your report.
Ping @PrestaShop/prestashop-product-team what do you think?

Thanks!

**Comment by vvvamik** [comment]
> PrestaShop/prestashop-product-team

Ho @khouloudbelguith 
Page PrestaShop/prestashop-product-team not works.

**Comment by khouloudbelguith** [comment]
Hi @vvvamik,

In your case, you don't have access to this link, but it is OK for the members of PrestaShop.

Thanks!

**Comment by LouiseBonnard** [comment]
Hello @vvvamik,

thanks for bringing up this issue we will need to study! Could you please add it to the [User Voice](http://feedback.prestashop.com/forums/387864-prestashop-1-7-x) so that the community can vote for it if they consider this idea useful? I will help us realize how requested is this feature and, why not, add it to the roadmap. ;-)
